### PR TITLE
Implement login and registration screens in react native

### LINKED ISSUE #7: Implement Login and Registration Screens in React Native
## REQUIREMENTS / DESCRIPTION

> This issue involves implementing the login and registration screens in a React Native application. The design for these screens has been provided, and the task is to translate the design into functional components and integrate them into the existing React Native project.

```[tasklist]
### Tasks
- [x] Convert the provided design Login Screen into a React Native component.
- [x] Implement the necessary input fields for email and password.
- [x] Add a login button.
- [x] Convert the provided design Registration Screen into a React Native component.
- [x] Implement the necessary input fields for name, email, password, and any other required fields.
- [x] Add a registration button.
```

### PR DESCRIPTION
**Pull request by Eduardoespl**
# Description

This pull request introduces the initial implementation of login and registration screens in React Native for our project. Please note that these screens are currently not fully functional but provide the necessary foundation for further development.

## Changes Made

* Created new Login and Register screens using React Native Components.
* Utilized appropiate React Navigation for seamless screen transitions.
* Implemented basic navigation between the login and registration screens.

It's important to highlight that while the screens are visually present and navigable, further functionality and integration with backend APIs are still pending. Future iterations will focus on completing the backend integration for user authentication and account creation.

Please review the code changes thoroughly and provide any feedback or suggestions for improvement.

## Screenshots

image

image

Grabación 2023-06-23 174802

### CODE REVIEW

**Review by montsegv-2 — APPROVED** [review]
LGTM!

**Review by Alfredo20164920 — APPROVED** [review]
Good Job 💯

File: AppVuelos/App.tsx [hunk 1]
```diff
@@ -6,113 +6,25 @@
...
+function App() {
   return (
-    <View style={styles.sectionContainer}>
-      <Text
-        style={[
-          styles.sectionTitle,
-          {
-            color: isDarkMode ? Colors.white : Colors.black,
-          },
-        ]}>
```
Comment (YilKanda) [inline AppVuelos/App.tsx:42]: The styles go in a separate file.

File: AppVuelos/components/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    marginTop: 5,
+    paddingHorizontal: 10,
+    paddingVertical: 5,
+  },
+  label: {
+    color: '#929292',
+    fontSize: 18,
+    fontWeight: '600',
+  },
+});
```
Comment (YilKanda) [inline AppVuelos/components/Input.tsx]: The styles go in a separate file.

File: AppVuelos/components/buttons.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    textAlign: 'center',
+    fontSize: 20,
+    justifyContent: 'space-between',
+  },
+  textNormal: {
+    color: '#929292',
+    textAlign: 'center',
+    fontSize: 16,
+  },
+});
```
Comment (YilKanda) [inline AppVuelos/components/buttons.tsx]: The styles go in a separate file.

File: AppVuelos/screens/login.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    fontSize: 16,
+    marginTop: 20,
+  },
+  TextLoginLink: {
+    color: '#5C6EF8',
+    textAlign: 'center',
+    fontSize: 16,
+    marginTop: 20,
+  },
+});
```
Comment (YilKanda) [inline AppVuelos/screens/login.tsx]: The styles go in a separate file.

You can create a specific library for colors.

File: AppVuelos/screens/register.tsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    fontSize: 16,
+    marginTop: 20,
+  },
+  TextLoginLink: {
+    color: '#5C6EF8',
+    textAlign: 'center',
+    fontSize: 16,
+    marginTop: 20,
+  },
+});
```
Comment (YilKanda) [inline AppVuelos/screens/register.tsx]: The styles go in a separate file.

You can create a specific library for colors.

**Review by YilKanda — DISMISSED** [review]

File: AppVuelos/components/Input.tsx (hunk 2 above)
Comment (Eduardoespl) [reply AppVuelos/components/Input.tsx]: Changes made

File: AppVuelos/components/buttons.tsx (hunk 3 above)
Comment (Eduardoespl) [reply AppVuelos/components/buttons.tsx]: Fixed this problem

File: AppVuelos/screens/login.tsx (hunk 4 above)
Comment (Eduardoespl) [reply AppVuelos/screens/login.tsx]: Changes made on screen login.tsx, created a file for the styles and a color library

File: AppVuelos/screens/register.tsx (hunk 5 above)
Comment (Eduardoespl) [reply AppVuelos/screens/register.tsx]: Changes made

File: AppVuelos/App.tsx (hunk 1 above)
Comment (Eduardoespl) [reply AppVuelos/App.tsx:42]: This file don't have styles on the recent commit

File: AppVuelos/screens/login.tsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+import {Link} from '@react-navigation/native';
+import Buttons from '../components/buttons';
+import loginStyles from '../styles/LoginStyles';
+
+function Login(): JSX.Element {
+  return (
+    <SafeAreaView style={loginStyles.container}>
+      <View style={loginStyles.rowContainer}>
+        <Text style={loginStyles.title}>Flight Bookings</Text>
+        <FontAwesome name="plane" size={30} color={'#5C6EF8'} />
```
Comment (toledodavid) [inline AppVuelos/screens/login.tsx:15]: Probably you should use primaryColor constant instead of hex value directly

**Review by toledodavid — APPROVED** [review]

**Comment by yovanha** [comment]
LGTM!

**Review by yovanha — APPROVED** [review]
LGTM!

File: AppVuelos/screens/login.tsx (hunk 6 above)
Comment (cfcortesmx) [reply AppVuelos/screens/login.tsx:15]: Done

**Pull request merged**